lib/compat/wordpress-6.6/resolve-patterns.php

```diff
@@ -71,6 +71,9 @@ function gutenberg_replace_pattern_blocks_get_block_templates( $templates ) {
 }
 
 function gutenberg_replace_pattern_blocks_get_block_template( $template ) {
+	if ( null === $template ) {
+		return $template;
+	}
```

Comment:
To me that sounds like there's a bug in the `get_block_template` filter, which should be called at all for `null`.